azure-pipelines-release.yml

```diff
@@ -21,28 +21,44 @@ jobs:
         osx - python37:
           image.name: 'macos-12'
           python.version: '3.7'
+          ciwb.build: cp37-macosx_x86_64
+          ciwb.archs: x86_64
         osx - python38:
           image.name: 'macos-12'
           python.version: '3.8'
+          ciwb.build: cp38-macosx_x86_64
+          ciwb.archs: x86_64
         osx - python39:
           image.name: 'macos-12'
           python.version: '3.9'
+          ciwb.build: cp39-macosx_universal2
+          ciwb.archs: universal2
         osx - python310:
           image.name: 'macos-12'
           python.version: '3.10'
+          ciwb.build: cp310-macosx_universal2
+          ciwb.archs: universal2
 
         windows - python37:
           image.name: 'windows-2019'
           python.version: '3.7'
+          ciwb.build: cp37-win_amd64
+          ciwb.archs: AMD64
```

Comment:
Won't this make it so that Windows will only have AMD64 builds? I've never worked with `cibuildwheel` before, but the documentation seems to indicate that specifying this flag will only build it for AMD64. I think having a x86 build would be pretty important?

Reply:
> Won't this make it so that Windows will only have AMD64 builds? 

Yes. But this matches the existing build configuration.